bin/compile

```diff
@@ -28,4 +28,8 @@ mkdir -p $BUILD_DIR/bin
 cp "$BUILDPACK_DIR/bin/run-stunnel" $BUILD_DIR/bin/
 chmod +x $BUILD_DIR/bin/run-stunnel
 
+echo "-----> Moving the run-stunnel default certificate ca into app/config"
```

Comment:
[nit] technically this cert is used by both `start-stunnel` and `run-stunnel`. I'd suggest just:
```
echo "-----> Moving the default CA into app/config"
```